events/wc_frostwolf_events.txt

```diff
@@ -58,11 +89,11 @@ narrative_event = {
 		}
 	}
 	option = {
-		name = EVTOPTB_WCFW.1
-		custom_tooltip = { text = EVTDESB_WCFW.1 }
+		name = We will be heading south.
```

Comment:
Typo.

Reply:
Good spot, probably a bad copy-paste there. This should be resolved.